pkg/ui/workspaces/cluster-ui/src/insights/workloadInsights/util/insightsColumns.tsx

```diff
@@ -238,7 +238,8 @@ export const insightsTableTitles: InsightsTableTitleType = {
   },
   cpu: (_: InsightExecEnum) => {
     return makeToolTip(
-      <p>{`CPU Time spent executing within the specified time interval.`}</p>,
+      <p>{`SQL CPU Time spent executing within the specified time interval. It
+      excludes the time spent in the KV level as well as SQL planning time.`}</p>,
```

Comment:
Alex's suggestion in [slack](https://cockroachlabs.slack.com/archives/C063CP41TG9/p1702510571290669?thread_ts=1702504100.144489&cid=C063CP41TG9) sounds clearer:

```suggestion
      does not include SQL planning time nor KV execution.`}</p>,
```